tests/production_restart.py

```diff
@@ -64,28 +65,46 @@
     cluster.biosNode.kill(signal.SIGTERM)
     cluster.waitOnClusterSync(blockAdvancing=5)
 
-    node0 = cluster.getNode(0) # producer
-    node1 = cluster.getNode(1) # finalizer
+    producerNode = cluster.getNode(0)
+    finalizerNode = cluster.getNode(1)
```

Comment:
These names (and in fact the original comments) are misleading because each node has its own producer and finalizer configured.

Reply:
Renamed.